THIRD-PARTY

```diff
@@ -213,8 +214,622 @@ limitations under the License.
 
 ------
 
-** openssl -- https://www.openssl.org/
-Copyright (c) 1998-2021 The OpenSSL Project
+** GnuPG; version 2.2.27 -- https://gnupg.org/
```

Comment:
Is it reasonable to remove the version number here as well?

Reply:
Since we need to link to the source code (I had to upload the original source to https://amazon-source-code-downloads.s3.amazonaws.com/aws/thinkbox/gnupg/gnupg-2.2.27.tar.bz2 as a part of the distribution request, this was a requirement for GPL) I think we should probably leave this one as-is. I will write up a short wiki or maybe a README alongside the script I wrote to upload the GPG installer to our buckets in the RFDK repository that explains the update process.

Reply:
Sounds good.